semantics.json

```diff
@@ -515,6 +515,13 @@
         "description": "Defines how many words that will be visible each time. If 0 is set, all words will be shown.",
         "type": "number",
         "optional": true
+      },
+      {
+        "label": "Enable multiple pages",
```

Comment:
Adding non-optional fields to `semantics.json` needs to be accompanied by sanitizing those values in the constructor (and potentially setting them for existing content via `upgrades.js`) or existing content will not know them. Here it will render `undefined`, but content can also crash in other cases.